savant/tests/test_basic.py

```diff
@@ -32,19 +34,10 @@ def test_compare(self):
         })
         self.assertEqual(response.status_code, 200)
 
-    def test_nullcases(self):
-        engine_name = "postgresql://postgres:postgres@127.0.0.1:5432/travis_ci_test"
-        conn = psycopg2.connect(engine_name)
-        cur = conn.cursor()
-
-        cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
-        conn.commit()
-        cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
-        conn.commit()
-        cur.execute("select count(*) from debs where stat=%s and name is null and version=%s and architecture=%s", ('stat', 'test', 'test'))
-        count = cur.fetchone()
-        self.assertEqual(count[0], 1)
-
-
+    #def test_nullcases(self):
```

Comment:
I feel like I'm doing github wrong. I saw this set of changes on an earlier commit, but this was already fixed. Very strange. Is this PR still active, or should we close it?

Reply:
I think it's my fault. Probably shouldn't have created a PR yet.

Reply:
@greenmoss I don't think this PR is active. can we delete this?

Reply:
Just realized it now